### DISCUSSION TITLE
Welcome to DiscordQt Discussions!

### DISCUSSION THREAD

**Discussion by ruslang02** [ruslang02/discord-qt#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by TaiAurori** [comment]
Are there any plans on adding folder functionality to DiscordQt?

**Reply by ruslang02** [reply]
It shouldn't be difficult to implement, all the properties and methods are there, but I won't be able to do it myself currently.

**Reply by ruslang02** [reply]
You could try implementing this feature yourself, would be cool!

**Reply by TaiAurori** [reply]
I don't plan to currently, but it's not off the table :wink:
